webpages/error/export.js

```diff
@@ -0,0 +1,4 @@
+document.querySelector("#export-settings-btn").onclick = () => {
+  // TODO
```

Comment:
Note to self: it's possible chrome.storage will be broken here, so settings won't be able to be exported.

Related: #6448